test/Interop/Cxx/value-witness-table/custom-destructors-irgen.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir | %FileCheck %s
```

Comment:
Assuming you rename the other test to `custom-destructors-virtual-irgen.swift`, please rename this one to `custom-destructors-non-virtual-irgen.swift`.

Reply:
I'm going to do the same thing with `custom-destructors.swift` -> `custom-destructors-non-virtual.swift`